api/sheets/get_name.md

```diff
@@ -24,23 +24,23 @@ Before trying this method out, make sure to follow the setup directions in [the
 
```

Comment:
If there's another outstanding PR for this file, can we remove it from this PR please?